scripts/install/opt/seagate/cortx/motr/conf/examples/provisioner_cluster.json

```diff
@@ -0,0 +1,310 @@
+{

```

Comment:
path is changed, same has to be updated in wiki as well and the old file needs to be deleted

Reply:
Wiki changed for absolute path and new config